### PR TITLE
Load config as valid JSON string in case it's not

### LINKED ISSUE #42: Validation fails with "No JSON object could be decoded" when trying to load conifg
I'm not sure why exactly this is happening, but I'm getting `No JSON object could be decoded` when validation jobs are run.
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/rq/worker.py", line 588, in perform_job
    rv = job.perform()
  File "/usr/lib/python2.7/site-packages/rq/job.py", line 498, in perform
    self._result = self.func(*self.args, **self.kwargs)
  File "/srv/app/src/ckanext-validation/ckanext/validation/jobs.py", line 47, in run_validation_job
    options = json.loads(options)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```

- CKAN version: 2.7.5
- Validation version 0.0.7
- `CKANEXT__VALIDATION__DEFAULT_VALIDATION_OPTIONS='{"row_limit": 100000}'`
- And this is **not** happening for all instances that have same exact configurations. 

### Few things I've tried to debug this:

- When I'm logging `options` and `type(options)` on line 47 (from above), I'm getting `'{"row_limit": 100000}'` and `<type 'str'>`
  - `json.loads({"row_limit": 100000}')` should be wokring and is working, but:
- According to the above I assumed that `options` equals to `'{"row_limit": 100000}'` and it's string, but it does not seem so, as `eval(options)` and `type(eval(options))` are giving `{"row_limit": 100000}` and `<type 'str'>` while it shoul be `<type 'dict'>`
  - however if I hardcode `json.loads(eval(options))` the error  disspars and validation is succeded .
- We are getting options from config like this https://github.com/frictionlessdata/ckanext-validation/blob/master/ckanext/validation/jobs.py#L40-L41
```
options = t.config.get(u'ckanext.validation.default_validation_options')
```
- If I hardcode `options='{"row_limit": 100000}'` there and repeat above, now I'm getting `<type 'dict'>` as it should be. and error dissaperas in this case as well (note I'm just logging `eval(options)` here - not asigning to any variables)

### Possible solution

I'm not sure what is exactly returned by `t.config.get()` that looks like string and is the type of `str`, but `json.loads()` is complaining about it anyway... Maybe it's `core` bug but as a quick solution (that will not harm anybody or anything) would be to check if `eval` still returns the `str` and if so pass to `json.loads()` that string.  Something similar to

```
if options:                                            
        if isinstance(eval(options), str):                 
            options=eval(options)                          
        options = json.loads(options) 
```

I'll push the PR shortly for review

### PR DESCRIPTION
**Pull request by zelima**
make sure config is loaded as a valid JSON string. fixes #42

### CODE REVIEW

File: ckanext/validation/jobs.py [hunk 1]
```diff
@@ -40,6 +40,11 @@ def run_validation_job(resource):
     options = t.config.get(
         u'ckanext.validation.default_validation_options')
     if options:
+        # Somtimes json.loads(options) fails with No JSON object could be decoded
+        # When that is the case eval(options) returns string, otherwise dict
+        # see https://github.com/frictionlessdata/ckanext-validation/issues/42
+        if isinstance(eval(options), str):
```
Comment (roll) [inline ckanext/validation/jobs.py]: I wasn't able to understand this issue but I would suggest considering to use `ast.literal_eval` instead of `eval`.

Comment (zelima) [reply ckanext/validation/jobs.py]: Instead of `json.loads()` right? Agree, just wanted to touch the original code as less as possible...

Comment (zelima) [reply ckanext/validation/jobs.py]: instead of `eval`...

**Comment by zelima** [comment]
INVALID see related issue

**Pull request closed without merging**